### PR TITLE
Update Autodesk Construction Cloud docs

### PR DESCRIPTION
**Pull request by joefields**
Hello Microsoft team, we would like to update our documentation based on our updated server regions as well as a few other minor edits.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@DougKlopfenstein 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by DougKlopfenstein — APPROVED** [review]

**Comment by DougKlopfenstein** [comment]
#sign-off

File: powerquery-docs/connectors/autodesk-construction-cloud.md [hunk 1]
```diff
@@ -64,7 +64,10 @@ To connect to Autodesk Construction Cloud data:
   
 ## Connect using Autodesk provided Power BI Templates
 
-* Download the latest Power BI Templates here: [https://construction.autodesk.com/templates/power-bi/](https://construction.autodesk.com/templates/power-bi/).
+* Download the latest Power BI Templates from the Template Gallery within **Data Connector**.
+
+  :::image type="content" source="./media/autodesk-construction-cloud/acc-powerbi-template-gallery.png" alt-text="Power BI Template Gallery in Data Connector":::
```
Comment (Jak-MS) [inline powerquery-docs/connectors/autodesk-construction-cloud.md:69]:
```suggestion
  :::image type="content" source="./media/autodesk-construction-cloud/acc-powerbi-template-gallery.png" alt-text="Screenshot showing the Power BI Template Gallery in Data Connector":::
```

**Comment by Jak-MS** [comment]
@joefields - thanks for your update. Microsoft style for screenshot callout is to use red boxes (as opposed to arrows).  We'll merge this today, but can you open a new PR with updated screenshots with boxes instead of arrows? Thanks!

**Pull request merged**